migration/index.js

```diff
@@ -0,0 +1,110 @@
+import { ApiPromise, WsProvider } from "@polkadot/api";
+import { readFileSync, writeFileSync } from "fs";
+import { Keyring } from "@polkadot/keyring";
+import { waitReady } from "@polkadot/wasm-crypto";
+
+const lite_node = "wss://main.mirror.test.opentensor.ai:443";
+const provider = new WsProvider(lite_node);
+const api = new ApiPromise({ provider: provider });
+
+const emit_map = readFileSync("emit_map_migration.json", "utf-8");
+const emit_map_json = JSON.parse(emit_map);
+
+const BATCH_MAX_SIZE = 1000;
+
+// TODO: Uncomment and fill in mnemonic
+// const mnemonic = "your mnemonic here"
+
+// TODO: fill with hotkey
+// const HOTKEY = ""
+
+// TODO: fill in your signer-key address
+// const SIGNER_KEY = ""
+
+// TODO: fill multisig key address
+// const MULTISIG_KEY = ""
+
+// TODO: fill in multi-sig signers
+// const signers = []
+
+const main = async (emit_map_json) => {
+  await waitReady();
+
+  // const wallet_key = new Keyring({ type: 'sr25519' }).addFromMnemonic(mnemonic)
+
+  const keyring = new Keyring({ type: "sr25519" });
+  const pub_key = keyring.addFromAddress(SIGNER_KEY);
+  signers.forEach((signer) => {
+    if (!!keyring.getPair(signer)) {
+      keyring.addFromAddress(signer);
+    }
+  });
+
+  await api.isReady;
+
+  let batches = [];
+  let batch_calls = [];
+  let curr_batch_size = 0;
+  console.log("Creating batches of calls");
+  Object.keys(emit_map_json).forEach(async (key) => {
+    if (curr_batch_size >= BATCH_MAX_SIZE) {
+      // Batch into BATCH_MAX_SIZE txs for block size reduction
+      batches.push(batch_calls);
+      batch_calls = [];
+      curr_batch_size = 0;
+    }
+
+    let to_emit = emit_map_json[key];
+    let tx = api.tx.balances.transferKeepAlive(key, to_emit);
+    batch_calls.push(tx);
+    curr_batch_size += 1;
+  });
+
```

Comment:
```suggestion
batches.push(batch_calls)
```
This is to push the last batch_calls into batches if they are less than BATCH_MAX_SIZE